mne/encoding/model.py

```diff
@@ -1,28 +1,66 @@
 import numpy as np
 from sklearn.base import BaseEstimator
 from scipy import linalg
+from .feature import EventsBinarizer, DataDelayer
 from ..externals.six import string_types
+from ..utils import warn
+from sklearn.base import is_regressor
+
+
+class EventRelatedRegressor(object):
+    def __init__(self, raw, events, est=None, event_id=None, tmin=-.5, tmax=.5,
```

Comment:
do we want to have tmin == tmax by default? I would have thought that by default we don't really want to look at t<0 (except that it shows the stats baseline)